src/Ryujinx.Graphics.Nvdec.Vp9/LoopFilter.cs

```diff
@@ -55,8 +54,7 @@ internal static class LoopFilter
         //    00000000
         //
         // A loopfilter should be applied to every other 4 the row vertically.
-        private static readonly ulong[] Above64X64TxformMask = new ulong[]
-        {
+        private static readonly ulong[] _above64X64TxformMask = {
             0xffffffffffffffffUL,  // TX_4X4
```

Comment:
```suggestion
            0xffffffffffffffffUL, // TX_4X4
```
Same for other comments